src/mixins/itext_key_behavior.mixin.js

```diff
@@ -116,11 +120,24 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
       return;
     }
     var keyMap = this.direction === 'rtl' ? this.keysMapRtl : this.keysMap;
-    if (e.keyCode in keyMap) {
-      this[keyMap[e.keyCode]](e);
+    var func;
+    if (e.key in keyMap) {
+      func = keyMap[e.key];
+    }
+    else if (e.keyCode in keyMap) {
+      func = keyMap[e.keyCode];
+    }
+    else if ((e.key in this.ctrlKeysMapDown) && (e.ctrlKey || e.metaKey)) {
```

Comment:
Shouldn't the ctrl maps come before the general key maps?

Reply:
eventually yes. Probably there was not overlapping?

Reply:
No overlapping in fabric. I had in mind the dev wanting to override what happens when a ctrl modifier is active.
Current logic doesn't allow.